lib/tasks/evm_application.rb

```diff
@@ -68,17 +68,8 @@ def self.status(include_remotes = false)
 
   def self.output_status(data, footnote = nil)
     return if data.blank?
-    duplicate_columns = redundant_columns(data)
```

Comment:
Can we remove `redundant_columns` method below?  Was this the only code calling it?

Reply:
aah - looks to be reimplemented over in ext_management_system

(darn, wish that code was in evm_application)